ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -80,6 +80,33 @@ interface {{ port_channel_interface }}
 {%         if port_channel_interfaces[port_channel_interface].rt is arista.avd.defined %}
       route-target import {{ port_channel_interfaces[port_channel_interface].rt }}
 {%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.redundancy is arista.avd.defined %}
+      redundancy {{ port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.redundancy }}
+{%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.designated_forwarder_election is arista.avd.defined %}
+{%             set designated_forwarder_cli = "designated-forwarder election" %}
```

Comment:
The reuse of this string across multiple sections below just makes it harder to read with no advantage. Please just set the full string in line 89, 96, 103.

Reply:
Will update line 89, 96, 103 as follows:

```yaml
{%                set designated_forwarder_algorithm_cli = "designated-forwarder election algorithm " ~ port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.designated_forwarder_election.algorithm ~ " " ~ port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.designated_forwarder_election.preference_value %}
{%                set designated_forwarder_hold_time_cli = "designated-forwarder election hold-time " ~ port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.designated_forwarder_election.hold_time %}
{%                set designated_forwarder_candidate_cli = "designated-forwarder election candidate reachability required" %}
```